frontend/src/views/LabelView.vue

```diff
@@ -79,4 +80,55 @@ watchEffect(() => {
       />
     </div>
   </v-container>
+  <v-container fluid class="d-flex align-center justify-center my-2">
+    <v-row justify="center">
+      <v-dialog v-model="dialog">
+        <template v-slot:activator="{ props }">
+          <v-btn color="primary" rounded v-bind="props">
+            Frame: {{ imgIndex + 1 }} of {{ framesList.length }}
+          </v-btn>
+        </template>
+        <v-card>
```

Comment:
Can you add a toolbar above the `v-card`. To save you time I've already got the toolbar. I ended up creating it as I needed to test the styling, it was kinda glitchy at the moment but got it working properly.
```tsx
<v-toolbar color="primary" class="rounded-t">
  <v-toolbar-title>Select Frame</v-toolbar-title>
  <v-spacer />
  <v-btn icon="mdi-close" @click="dialog = false" />
</v-toolbar>
```

Additionally, to make the dialog expand to the window size, and hide the rounded corners at the top of the card you'll need to add the following and you can also remove the `v-card-title`.

```suggestion
        <v-card style="height: calc(100vh - 104px)" class="rounded-t-0">
```